engine/src/test/java/io/zeebe/engine/processor/workflow/deployment/CreateDeploymentTest.java

```diff
@@ -157,7 +157,9 @@ public void shouldCreateDeploymentResourceWithMultipleWorkflows() {
   public void shouldCreateDeploymentIfUnusedInvalidMessage() {
     // given
     final BpmnModelInstance process = Bpmn.createExecutableProcess().startEvent().done();
-    process.getDefinitions().addChildElement(process.newInstance(Message.class));
+    final Message message = process.newInstance(Message.class);
+    message.setName("message");
+    process.getDefinitions().addChildElement(message);
```

Comment:
I think that adding a name to the message is not in the intent of this test case ;) 

The idea was that a message without a name is invalid. But if the message is not used in the workflow (this can happen using the modeler) then it is ignored.

Reply:
Well, we are no stricter than that. 

```
        ZeebeExpressionValidator.verifyThat(Message.class)
            .hasValidExpression(
                Message::getName,
                expression -> expression.isConstantWithResultType(ResultType.STRING).isMandatory())
```

Not sure if I could figure out whether a message is used or not at this point. Or if the constraint in the runtime expression should be relaxed. Or if we cannot support this use case any longer.